src/reducers/books.js

```diff
@@ -0,0 +1,10 @@
+const books = (books = [], { type, payload }) => {
+  if (type === 'CREATE_BOOK') {
```

Comment:
- [ ] [OPTIONAL] A good architecture idea to avoid typos and make refactoring faster in the future is to save the constant strings to constants in a file. 
So you could have a file that contains only types of actions and then you can call the constants instead of the strings.